### DISCUSSION TITLE
selecting field for csv output

### DISCUSSION THREAD

**Discussion by tomboul** [hertzg/rtl_433_docker#66] [Q&A] [answered]
Hello

Excellent work !!
I spent two days trying solutions to integrate a RTL-433 on a QNAP, and yours worked the first time and very very quickly !!!

Is it possible to select fields for the CSV file? There are 357 fields, but I only need 8 fields :
"time", "model", "id" ,  "channel", "battery_ok", "temperature_F",,"humidity", "mic"
The json format is a solution because output give only not empty field, but i will get also big logs, they are the name of the field in each line.

For csv or json, there is always this warning message : bitbuffer_add_bit: Warning: row count limit (50 rows) reached

Otherwise, i use a cron task every 10 mn for running container with json output, so i get the json ouput and write value into a data base ....

Bravo !! the choice of container is the only portability solution for the general public use of linux

Bet regard
Thomas

**Comment by hertzg** [comment, marked as answer]
There are numerous ways to use the tool, depending on your usecase you will need to build around the tool to satisfy your specific needs. JSON with `jq` tool help you but you will need to set it up as it's a custom requirement.
